internal/juju/offers.go

```diff
@@ -197,17 +197,17 @@ func (c offersClient) DestroyOffer(input *DestroyOfferInput) error {
 	}
 
 	forceDestroy := false
-	//This code loops until it detects 0 connections in the offer or 3 minutes elapses
+	//This code loops until it detects 0 connections in the offer or 30s elapses
```

Comment:
issue: I disagree with being more aggressive here. There are a number of GitHub issues already where the provider is failing to wait for a resource to be destroyed. 

If this is taking a long time, it's most likely because we're not waiting for the integration to be properly removed. Though that does not improve the test times.

Reply:
I've reverted this bit and kept the other change.